### DISCUSSION TITLE
[Run issue]:TOPAS Graphical User Interface (GUI) error：“topas”is not responding.

### DISCUSSION THREAD

**Discussion by chuanye90** [OpenTOPAS/OpenTOPAS#92] [Run-Problem] [answered]
### Contact Details

liuchuanye90@buaa.edu.cn

### What was the issue?

Dear expert:

        The following error occurred while I was running the simulation in the topas GUI interface.
        If I click on "Wait," it returns to normal, but the error appears again after a few seconds.
        However, this error does not affect the simulation operation, unless you click 'Force Quit' to stop the simulation.
        Thus, I'm not sure whether this error is caused by a software issue or a computer problem.
![Uploading Screenshot from 2024-10-28 16-01-12.png…]()

### TOPAS category

Physics

### Version

3.9

### What Operative System (OS) are you running on?

Linux

### Code of Conduct

- [X] I agree to follow this project's Code of Conduct

### Attestation

- [X] I testify that I have fully read the bug reporting guideline beforehand.

**Comment by chuanye90** [comment]
![Screenshot from 2024-10-28 16-01-12](https://github.com/user-attachments/assets/29d1938e-407f-42ca-9f08-07eb4d6fdbb0)

**Reply by JoseRamosMendez** [reply, marked as answer]
Hi there,
The GUI is for prototyping and developing the simulations. Running many particle tracks causes the GUI to freeze. You can try with just a few tracks, or by displaying only specific particles. See [here](https://opentopas.readthedocs.io/en/latest/parameters/graphics.html)
Best.
